.github/workflows/release-test.yml

```diff
@@ -0,0 +1,126 @@
+name: Release-test
+on: [pull_request]
+# use pull request for test, will change to push before merge
+  #push:
+  #  branches:
+  #    - '*'
+  #    - '!master'
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - ubuntu-18.04
+        os_name:
+          - linux_openresty
+          - linux_openresty_1_17
+          - linux_openresty_1_15
+          - linux_tengine
+          - linux_apisix_master_luarocks
+          - linux_apisix_current_luarocks
+          - linux_openresty_mtls
+
+    runs-on: ${{ matrix.platform }}
+    env:
+      SERVER_NAME: ${{ matrix.os_name }}
+      OPENRESTY_VERSION: default
+
+
+    services:
+      etcd:
+        image: bitnami/etcd:3.4.0
+        ports:
+          - 2379:2379
+          - 2380:2380
+        env:
+          ALLOW_NONE_AUTHENTICATION: yes
+          ETCD_ADVERTISE_CLIENT_URLS: http://0.0.0.0:2379
+
+      old_etcd:
```

Comment:
There is too much repeated code. Can we reduce them? Or can we just modify the `linux_openresty_common_runner.sh`?

Reply:
Maybe we can add a new step in the build.yml, and remove the `apisix` in git repo, replace `./bin/apisix` with `/usr/bin/apisix`? So thev remain don't need to be changed.